From 784d8ff89e1771a5592b38989f5253fa53536cba Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Mon, 23 Jul 2018 16:07:40 -0700
Subject: [PATCH] t6100--mdraid-partitions: Use v0.90 metadata for the test

Newer metadata types use more disk space, causing the test to fail.
---
 tests/t6100-mdraid-partitions.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/t6100-mdraid-partitions.sh b/tests/t6100-mdraid-partitions.sh
index dbb1686..f051589 100755
--- a/tests/t6100-mdraid-partitions.sh
+++ b/tests/t6100-mdraid-partitions.sh
@@ -45,8 +45,8 @@ cleanup_fn_() {
   mdadm -S $md_dev || warn_ "Failed to stop MD array, $md_dev"
 }
 
-# create mdraid on top of both partitions
-mdadm -C $md_dev --force -R -l1 -n2 "${scsi_dev}1" "${scsi_dev}2"
+# create mdraid on top of both partitions with v0.90 metadata
+mdadm -C $md_dev -e0 --force -R -l1 -n2 "${scsi_dev}1" "${scsi_dev}2"
 
 # create gpt and two partitions on the raid device
 parted -s $md_dev mklabel gpt \
-- 
2.17.1

